dhis-2/dhis-support/dhis-support-db-migration/src/main/resources/org/hisp/dhis/db/migration/2.43/V2_43_5__alter_primary_key_min_max_dataelement.sql

```diff
@@ -0,0 +1,29 @@
+-- Migration script to change the primary key of the minmaxdataelement table
+-- Make the surrogate key (minmaxdataelementid) a sequence
+ALTER TABLE minmaxdataelement
```

Comment:
Can't we skip this and then drop this old PK column at the end of the script?

Reply:
I am not sure we can, due to the various hibernate methods which I think may depend on it. My suggestion would be that maybe we handle this in a new PR, namely to reuse the bulk methods for the single value API  and then worry about removing it.